### PR TITLE
Fix bugzilla 1370 and 1371 for WAFS blending job

### PR DESCRIPTION
**Pull request by YaliMao-NOAA**
To address issue #63 and #64 
1. change variable name 'maillist' to 'MAILTO'
2. assign the value in job cards instead of in scripts

### CODE REVIEW

**Review by KateFriedman-NOAA — APPROVED** [review]
Looks good, thanks @YaliMao-NOAA !

**Comment by YaliMao-NOAA** [comment]
@KateFriedman-NOAA Thank you!

File: dev/driver/run_JWAFS_GRIB2_0P25_BLENDING [hunk 1]
```diff
@@ -83,16 +83,24 @@ export KEEPDATA=YES
...
 
 export DATA=/lfs/h2/emc/ptmp/$USER/working_wafs/blending.$jobid
 
 ############################################
 # run the job
 #############################################
+# Set up mailing list
+if [[ "${envir}" != "prod" ]]; then
+    MAILTO="yali.mao@noaa.gov"
+fi
```
Comment (aerorahul) [inline dev/driver/run_JWAFS_GRIB2_0P25_BLENDING:101]: Please don't hard-wire user emails/

File: ecf/scripts/grib2/0p25/blending/jwafs_grib2_0p25_blending_master.ecf [hunk 2]
```diff
@@ -40,6 +40,12 @@ set -x
 ############################################################
 # CALL executable job script here
 ############################################################
+# Set up mailing list
+if [[ "${envir}" != "prod" ]]; then
+    MAILTO="nco.spa@noaa.gov"
+fi
```
Comment (aerorahul) [inline ecf/scripts/grib2/0p25/blending/jwafs_grib2_0p25_blending_master.ecf:46]: What is this block supposed to achieve.
Does `nco.spa` need to be alerted for a parallel, or a test?

**Pull request merged**